duckduckgo/macos-browser
# Bump find-in-page version

Task/Issue URL: https://app.asana.com/0/1199230911884351/1203372521327064/f
Tech Design URL:
CC:

**Description**:
Bumps find-in-page to latest version to include bugfix



**Steps to test this PR**:
1. Go to https://refactoring.guru/refactoring/catalog
2. Perform a find
3. Confirm works as expected

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
